templates/dataset-detail.php

```diff
@@ -195,3 +195,9 @@ function render_metadata_table($supported_fields,$data){
 }
 
 ?>
+<script>
+  //Send GA Event
+  jQuery(function(){
+    ga('send', 'event', 'Dataset', 'view', '<?php echo $dataset_id; ?>');
```

Comment:
GA analytic JS is already included by GA plug-in we are using. So i think we don't need to include it again. 
I can't test it properly from local. Is it possible to implement it on PP and test it ?